### PR TITLE
Fix oracle database field

### PR DESCRIPTION
**Pull request by paulozulato**
# targeting `release-50`

Fixes https://emqx.atlassian.net/browse/EMQX-9980

As Oracle was not released yet, schema changes are backward compatible.

### <samp>🤖 Generated by Copilot at 66bed6d</samp>

This pull request fixes and improves the Oracle and Pulsar bridges and connectors, by adding new configuration options, fixing typos and bugs, and updating the dependencies and i18n files. It also adds a `proxy_header` option to the dashboard listener configuration.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] ~Added tests for the changes~
- [ ] Changed lines covered in coverage report
- [ ] ~Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files~
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] ~If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up~
- [x] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] ~If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)~
- [ ] ~Change log has been added to `changes/` dir for user-facing artifacts update~

### CODE REVIEW

File: apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl [hunk 1]
```diff
@@ -196,7 +196,6 @@ oracle_config(TestCase, _ConnectionType, Config) ->
         io_lib:format(
             "bridges.oracle.~s {\n"
             "  enable = true\n"
-            "  database = \"~s\"\n"
```
Comment (thalesmg) [inline apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl:199]: Nit: maybe add `service_name` to assert it's a valid config as expected.

Comment (paulozulato) [reply apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl:199]: we use `service_name` for Oracle RAC, which I believe it's not supported by oracleinanutshell. Therefore, I'm setting only `sid`. Also, both `sid` and `service_name` are optionals, so missing `service_name` here is valid =)

Comment (thalesmg) [reply apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl:199]: I see.  I was suggesting adding `service_name` here to assert that it is a valid field.

File: apps/emqx_bridge_oracle/src/emqx_bridge_oracle.erl [hunk 2]
```diff
@@ -66,25 +66,27 @@ namespace() -> "bridge_oracle".
...
+                )},
+            {local_topic,
+                hoconsc:mk(
+                    binary(),
+                    #{desc => ?DESC("local_topic"), default => undefined}
+                )}
+        ] ++ emqx_resource_schema:fields("resource_opts") ++
+            (emqx_oracle_schema:fields(config) --
+                emqx_connector_schema_lib:prepare_statement_fields()),
+    proplists:delete(database, Fields);
```
[the remark this answers is not in the document]
Comment (paulozulato) [reply apps/emqx_bridge_oracle/src/emqx_bridge_oracle.erl:89]: actually, it's added here by the `emqx_oracle_schema:fields(config)`, which adds the `emqx_connector_schema_lib:relational_db_fields()`, which adds the `database`.

[the remark this answers is not in the document]
Comment (thalesmg) [reply apps/emqx_bridge_oracle/src/emqx_bridge_oracle.erl:89]: But `emqx_oracle_schema:fields(config)` already removes `database`, doesn't it?

https://github.com/emqx/emqx/blob/80605b375eff106289f33c14e113e6e74ca2f558/apps/emqx_oracle/src/emqx_oracle_schema.erl#L26

I didn't understand how it appears again here, sorry.

File: apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl (hunk 1 above)
Comment (thalesmg) [reply apps/emqx_bridge_oracle/test/emqx_bridge_oracle_SUITE.erl:199]: After discussing, I understood that we can't test this field with the current oracle test container.

File: apps/emqx_bridge_oracle/src/emqx_bridge_oracle.erl (hunk 2 above)
[the remark this answers is not in the document]
Comment (paulozulato) [reply apps/emqx_bridge_oracle/src/emqx_bridge_oracle.erl:89]: oh, sorry. My bad. You're right.

**Review by zmstone — APPROVED** [review]

**Pull request merged**